modules/cloudsql-instance/variables.tf

```diff
@@ -204,7 +204,17 @@ variable "network_config" {
   }
 }
 
-variable "postgres_client_certificates" {
+variable "ssl_mode" {
+  description = "(Optional) Specify how SSL connection should be enforced in DB connections. This field provides more SSL enforcment options compared to require_ssl. To change this field, also set the correspoding value in require_ssl. For PostgreSQL instances, the value pairs are listed in the API reference doc for ssl_mode field. For MySQL instances, use the same value pairs as the PostgreSQL instances. For SQL Server instances, set it to ALLOW_UNENCRYPTED_AND_ENCRYPTED when require_ssl=false and ENCRYPTED_ONLY otherwise."
```

Comment:
Can your trim down this description a little bit? Maybe just one or two sentences and link to the provider field for more details.

Also, remove the "(Optional)" which is already implied by `default=null`